junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileSources.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2015-2024 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.params.provider;
+
+import static org.apiguardian.api.API.Status.STABLE;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.apiguardian.api.API;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@API(status = STABLE, since = "5.11")
+public @interface CsvFileSources {
+	CsvFileSource[] value();
```

Comment:
Please add Javadoc to the class and attribute similar to what's present on `org.junit.jupiter.api.Tags`.

Reply:
Java docs added for all new annotation containers. Thank you! I've missed them initially.
ref: f899d5b